src/css/components/_c-404.scss

```diff
@@ -11,7 +11,7 @@
 	background-color: $color-white;
 	border-top: $border-thicker solid $color-pink-shade;
 	margin-top: 2rem;
-	padding: 1rem 2rem 2rem 2rem;
```

Comment:
Looks like `padding-left` is getting dropped. Is this intentional?

Reply:
So, this is "intentional" but maybe I'm not understanding. If you're asking if the browser knows what to do, in the 3 value variant of the syntax, `padding-left` and `padding-right` are both set to the middle value.

If you are saying, you want to explicitly specify redundant values, we need to change the stylelint [rule `shorthand-property-no-redundant-values`](https://stylelint.io/user-guide/rules/shorthand-property-no-redundant-values) to allow for it. 

Can you advise which direction you prefer (removing the stylelint rule to allow redundant values or leaving this change as is)?